modules/distr-tracing-architecture.adoc

```diff
@@ -4,7 +4,7 @@ This module included in the following assemblies:
 -dist_tracing_arch/distr-tracing-architecture.adoc
 ////
 :_content-type: CONCEPT
-[id="distr-tracing-architecture_{context}"]
+[id="distributed-tracing-architecture_{context}"]
```

Comment:
According to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata, module IDs must match file names. 

Check other modules for the same issue.

Reply:
Argh, missed one!